src/exception/repo/entity.ts

```diff
@@ -135,13 +116,15 @@ const entityExceptionRepository: EntityExceptionRepository = {
         );
         entityExceptionDoc[entity] = filteredEntities as any;
         const doc = await entityExceptionDoc.save();
-        return checkDoc(doc);
-      } else {
-        return RepoError.DOCUMENT_UNDEFINED;
+        return doc;
       }
+      // mongo will return nulls for non existent docs
+      // tslint doesn't complain about a null from a lib
+      // tslint:disable-next-line
+      return null;
```

Comment:
this return is preceded by an `if / else`, so it looks like it'd be unreachable and can be removed
```suggestion
```

Reply:
just preceded by an `if`
this is catching if no doc is found

Reply:
ah!! 😆 I saw the removed else and mistook it as part of the active code.
apologies!